shogun-lib/src/main/resources/graphql/shogun.graphqls

```diff
@@ -1694,21 +1694,21 @@ type User implements BaseEntity {
     """
     clientConfig: JSON
     """
-    The user details stored in the associated Keycloak entity.
+    The user details stored in the associated provider details entity.
     """
-    keycloakRepresentation: UserRepresentation
+    providerDetails: UserRepresentation
```

Comment:
This is not the correct type for `User` types that have another generic type for `providerDetails`.

I discussed with @ahennr some other possibilities to do this with graphql, but these seem to be a little more complicated. (Especially not knowing which features our old graphql version supports).

I think your proposed change is a good intermediate solution! This would enable existing software projects to use a new shogun version without much rewriting until a better solution is found and implemented.